Switching libproxy and grnome_proxy to 'auto' instead of 'enabled'
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/glib-networking-2.80.0.orig/meson_options.txt b/glib-networking-2.80.0/meson_options.txt
index 5c40553..3e2e813 100644
--- a/glib-networking-2.80.0.orig/meson_options.txt
+++ b/glib-networking-2.80.0/meson_options.txt
@@ -7,7 +7,7 @@
 option('gnutls', type: 'feature', value: 'enabled', description: 'support for GnuTLS networking configuration')
 option('openssl', type: 'feature', value: 'disabled', description: 'support for OpenSSL networking configuration')
 option('environment_proxy', type: 'feature', value: 'disabled', description: 'support for environment variable proxy configuration')
-option('libproxy', type: 'feature', value: 'enabled', description: 'support for libproxy proxy configuration')
-option('gnome_proxy', type: 'feature', value: 'enabled', description: 'support for GNOME desktop proxy configuration')
+option('libproxy', type: 'feature', value: 'auto', description: 'support for libproxy proxy configuration')
+option('gnome_proxy', type: 'feature', value: 'auto', description: 'support for GNOME desktop proxy configuration')
 option('installed_tests', type: 'boolean', value: false, description: 'enable installed tests')
 option('debug_logs', type: 'boolean', value: false, description: 'enable debug log messages (slow)')
